internal/converter/testdata/wellknown.go

```diff
@@ -40,8 +40,12 @@ const WellKnown = `{
                     "format": "regex"
                 },
                 "struct": {
-                    "additionalProperties": true,
```

Comment:
Because it is now treated as a special case, the additionalProperties flag is not set (see line 265 and 270 in types.go)

Reply:
That's fine, we don't use this field in applied2.